src/caesar.js

```diff
@@ -0,0 +1,18 @@
+const caesar = (str, num) => {
+  let newMessage = '';
+  for (let i = 0; i < str.length; i += 1) {
```

Comment:
- [ ] [OPTIONAL] Did you know you can use a 'for of' loop with strings to get it's characters?
ex: `for (let char of 'fdgdf') { console.log(char) }`
Could you refactor your loop to use that?